bitnami/harbor/templates/chartmuseum/chartmuseum-dpl.yaml

```diff
@@ -0,0 +1,128 @@
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: http
+            {{- if .Values.chartmuseum.enableTLS }}
+            scheme: "HTTPS"
+            {{- end }}
+          initialDelaySeconds: {{ .Values.chartmuseum.readinessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.chartmuseum.readinessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.chartmuseum.readinessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.chartmuseum.readinessProbe.successThreshold }}
+          failureThreshold: {{ .Values.chartmuseum.readinessProbe.failureThreshold }}
+        {{- end }}
+        envFrom:
+        - configMapRef:
+            name: "{{ template "harbor.chartmuseum" . }}-envvars"
+        - secretRef:
+            name: "{{ template "harbor.chartmuseum" . }}-secret"
+        env:
+        - name: DEBUG
+          value: {{ ternary "1" "0" .Values.chartMuseumImage.debug | quote }}
+        {{- if .Values.extraEnvVars }}
+        {{ toYaml .Values.extraEnvVars | indent 8 }}
+        {{- end }}
+        ports:
+        - containerPort: {{ .Values.chartmuseum.port }}
+          name: http
+        volumeMounts:
+        - name: chartmuseum-data
+          mountPath: /bitnami/data
+        {{- if and .Values.persistence.enabled (eq .Values.persistence.imageChartStorage.type "gcs") }}
+        - name: gcs-key
+          mountPath: /etc/chartmuseum/gcs-key.json
+          subPath: gcs-key.json
+        {{- end }}
+        {{- if .Values.chartmuseum.enableTLS }}
+        - name: tls-certs
+          mountPath: /bitnami/certs
+        {{- end }}
+      volumes:
+      - name: chartmuseum-data
+      {{- if and .Values.persistence.enabled (eq .Values.persistence.imageChartStorage.type "filesystem") }}
+        persistentVolumeClaim:
+          claimName: {{ .Values.persistence.persistentVolumeClaim.chartmuseum.existingClaim | default (include "harbor.chartmuseum" .) }}
+      {{- else }}
+        emptyDir: {}
+      {{- end }}
+      {{- if and .Values.persistence.enabled (eq .Values.persistence.imageChartStorage.type "gcs") }}
+      - name: gcs-key
+        secret:
+          secretName: "{{ template "harbor.chartmuseum" . }}-secret"
+          items:
+            - key: GCS_KEY_DATA
+              path: gcs-key.json
+      {{- end }}
+      {{- if .Values.chartmuseum.enableTLS }}
+      - name: tls-certs
+        secret:
+          secretName: "{{ template "harbor.chartmuseum" . }}-secret"
+          items:
+            - key: TLS_CERT
+              path: server.crt
+            - key: TLS_KEY
+              path: server.key
+      {{- end }}
+  {{- with .Values.chartmuseum.nodeSelector }}
+  nodeSelector: {{ tpl (toYaml .) $ | nindent 8 }}
+  {{- end }}
+  {{- with .Values.chartmuseum.affinity }}
+  affinity: {{ tpl (toYaml .) $ | nindent 8 }}
+  {{- end }}
+  {{- with .Values.chartmuseum.tolerations }}
+  tolerations: {{ tpl (toYaml .) $ | nindent 8 }}
+  {{- end }}
```

Comment:
Move all these parameters: `tolerations`, `affinity` and `nodeSelector` after the `securityContext` and indent it properly